crates/agentgateway/src/proxy/httpproxy.rs

```diff
@@ -1134,6 +1145,59 @@ async fn make_backend_call(
 						.map_err(ProxyError::Processing)?;
 					(req, LLMResponsePolicies::default(), None)
 				},
+				RouteType::AnthropicTokenCount => {
+					let crate::llm::AIProvider::Bedrock(bedrock_provider) = &llm.provider else {
+						return Ok(Box::pin(async move {
+							Ok(
+								::http::Response::builder()
+									.status(::http::StatusCode::NOT_IMPLEMENTED)
+									.header(::http::header::CONTENT_TYPE, "application/json")
+									.body(http::Body::from(
+										r#"{"error":"Provider does not support Anthropic count_tokens API"}"#,
+									))
+									.expect("Failed to build response"),
+							)
+						}));
+					};
+
+					let mut req = crate::llm::bedrock::process_count_tokens_request(
+						bedrock_provider,
+						req,
+						llm_request_policies.llm.as_deref(),
+					)
+					.await
+					.map_err(|e| ProxyError::Processing(e.into()))?;
+
+					llm
+						.provider
+						.setup_request(&mut req, route_type, None, true)
+						.map_err(ProxyError::Processing)?;
+
+					auth::apply_late_backend_auth(
+						backend_call.backend_policies.backend_auth.as_ref(),
+						&mut req,
+					)
+					.await?;
+
+					let transport = build_transport(
```

Comment:
why do we need to do this here instead of falling through like we do for the rest of the route types?